src/test/java/seedu/address/testutil/TypicalEntries.java

```diff
@@ -100,6 +100,84 @@
             .withTags(VALID_TAG_SCIENCE, VALID_TAG_TECH)
             .build();
 
+    public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
+
+    // For ModelManagerTest
+    public static final String STUB_LINK_URL = "http://www.description.test/title/title_test.html";
```

Comment:
Mmm I see... but I'm not sure if `STUB_ENTRY_LINK` correctly describes it - it is the link URL that is the "stub" (which I mean a link that is a valid URL but is not real) rather than the entry.

Reply:
Ahh ok. In that case I guess we can leave it as that